django_ipam/templates/admin/django-ipam/subnet/change_form.html

```diff
@@ -2,6 +2,20 @@
 {% load i18n admin_urls %}
 {% load static %}
 
+{% block object-tools-items %}
+{{ block.super }}
+<li>
+  <a href="{% url 'admin:export_ipaddress' subnet.id %}">
+    {% trans "Export Ip address" %}
```

Comment:
this wording is a bit misleading: we say "Export Ip address" but in the truth we are exporting the entire subnet.
I think it should be: "Export Data" or simply "Export".